components/views/hrm/employees/employee-list/employeelist/Employeelist.jsx

```diff
@@ -101,6 +104,25 @@ export default function Employeelist() {
 		});
 	}
 
+	const onChange = (key, value) => {
+		const {filters={}} = state;
+
+		fetchEmployees({
+			...filters,
+			[key]: value,
+			page: name=='page' ? value : 1
+		});
```

Comment:
The problem of calling `fetchEmployees` right here is if the filter object gets changed from another function, then it will not fetch the list again. Rather we should put the filter data in filter state, and the `fetchEmployees` function should be called by `useEffect` hook. 

I see currently the `useEffect` dependency array is empty. You could have simply put the filter object in the array.

Reply:
Resolved.